packages/langium-vscode/data/langium.tmLanguage.json

```
@@ -25,13 +25,11 @@
         },
         {
             "name": "string.quoted.double.langium",
-            "begin": "\"",
-            "end": "\""
+            "match": "\"(\\\\.|[^\"\\\\])*\""
```

Suggestion: We should try to keep the `begin` and `end` properties, since the `match` properties only highlights full matches, not partial matches (i.e. lines which have a `"` in them but no end delimiter).

Add the following rule to the `repository` object:

```json
"string-character-escape": {
    "name": "constant.character.escape.ts",
    "match": "\\\\(x[0-9A-Fa-f]{2}|u[0-9A-Fa-f]{4}|u\\{[0-9A-Fa-f]+\\}|[0-2][0-7]{0,2}|3[0-6][0-7]?|37[0-7]?|[4-7][0-7]?|.|$)"
},
```

```suggestion
            "begin": "\"",
            "end": "\"",
            "patterns": [
				{
					"include": "#string-character-escape"
				}
			]
```